.circleci/README.md

```diff
@@ -0,0 +1,74 @@
+# Pytorch Serve CircleCI build
+TorchServe uses CircleCI for builds. This folder contains the config and scripts that are needed for CircleCI.
+
+## config.yml
+_config.yml_ contains TorchServe's build logic which will be used by CircleCI.
+
+## Workflows and Jobs
+Currently, following _workflows_ are available -
```

Comment:
Maintain existing names - torch-serve-build-gpu / torch-serve-build-cpu / torch-serve-regression-test-cpu / torch-serve-regression-test-gpu / torch-serve-regression-performance-test-cpu / torch-serve-regression-performance-test-gpu

Reply:
Workflow names revised to sanity, regression, performance